Scripts/Runtime/Jobs/Debug/BusyWorkJob.cs

```diff
@@ -0,0 +1,67 @@
+using Anvil.Unity.DOTS.Util;
+using Unity.Burst;
+using Unity.Jobs;
+
+namespace Anvil.Unity.DOTS.Jobs
+{
+    /// <summary>
+    /// An <see cref="IJob"/> job that will take up CPU time by calculating the Nth prime number passed in.
+    /// Useful for artificially increasing the time a job takes to help with debugging scheduling or profiling.
+    /// <seealso cref="DebugUtil.FindPrimeNumber"/>
+    /// </summary>
+    [BurstCompile]
+    public struct BusyWorkJob : IJob
```

Comment:
Looks good. Might want to consider an `IJobFor` so you can test parallel or single worker using the one job implementation.